### DISCUSSION TITLE
[Bug] copy link spam

### DISCUSSION THREAD

**Discussion by Miniontoby** [npm/feedback#883] [Website] [answered]
You can spam the copy link button and when scrolled a bit down it will fill up the whole top of your screen.

![image](https://user-images.githubusercontent.com/47940064/226118089-d63132a3-27c6-4dee-b69e-16e63dd55173.png)

**Comment by monishcm** [comment, marked as answer]
The message should dismiss itself in few seconds. I see the npmjs site in black. Do you think it is because of the plugin that changes themes causing it?

**Reply by Miniontoby** [reply]
> The message should dismiss itself in few seconds

I know the messages disappear after a few seconds, but I would just add a cooldown. It is not a huge problem tho, but it would be nice if it would not be possible to spam it.



> I see the npmjs site in black. Do you think it is because of the plugin that changes themes causing it?

Uhmm what are you talking about? I don't see any problems with the site's colors...
I just have force darkmode enabled in my browser causing all website to be dark'end and because the website itself doesn't have darkmode.

**Reply by monishcm** [reply]
This is your own action causing it right? I can take this into our backlog but will likely be very low on priority.
